crates/router/tests/connectors/utils.rs

```diff
@@ -962,6 +966,8 @@ impl Default for TokenType {
         let data = types::PaymentMethodTokenizationData {
             payment_method_data: types::api::PaymentMethodData::Card(CCardType::default().0),
             browser_info: None,
+            amount: 0,
+            currency: storage::enums::Currency::USD,
```

Comment:
If amount & currency is not required, please remove those from PaymentMethodTokenizationData

Reply:
These fields are added to generate AuthorizeSessionTokenData in Square Session Call